config/kcp/api_export.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: apis.kcp.dev/v1alpha1
+kind: APIExport
+metadata:
+  name: release
+spec:
+  latestResourceSchemas:
```

Comment:
`"9959e22acaaf6d162a03980c75b93dfe.releaseplans.appstudio.redhat.com" is forbidden: [metadata.name: Invalid value: "9959e22acaaf6d162a03980c75b93dfe.releaseplans.appstudio.redhat.com": must match ^[a-z]([-a-z0-9]*[a-z0-9])?$ in front of .releaseplans.appstudio.redhat.com]`

Reply:
We should update our patterns. Names can start with a number https://kubernetes.io/docs/concepts/overview/working-with-objects/names.

Reply:
I created a ticket for this: https://issues.redhat.com/browse/HACBS-1090